poly-commitment/benches/ipa.rs

```diff
@@ -8,6 +8,268 @@ use poly_commitment::{
     commitment::CommitmentCurve, ipa::SRS, utils::DensePolynomialOrEvaluations, PolyComm, SRS as _,
 };
 
+fn benchmark_msm(c: &mut Criterion) {
```

Comment:
Can you extract this in a separate PR? It does seem unrelated to the parallelization. It is a benchmark.